launchd/autorun_LaunchDaemon/com.googlecode.munki.autorun.plist

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>KeepAlive</key>
+	<dict>
+		<key>PathState</key>
+		<dict>
+			<key>/Users/Shared/.com.googlecode.munki.autorun</key>
+			<true/>
+		</dict>
+	</dict>
+	<key>Label</key>
+	<string>com.googlecode.munki.autorun</string>
+	<key>Program</key>
+	<string>/tmp/autorun.sh</string>
```

Comment:
This is really odd. You are installing a permanent launchdaemon that calls a script that will no longer be present after the next reboot. So after a reboot, we have a _broken_ or useless LaunchDaemon.

1) Why the need for a script? Why not just call the tool(s) directly?
2) If launchd is the right choice here, perhaps instead of installing a permanent launchd job for a one-time operation, maybe make use of `launchctl submit`?

Reply:
Ok, I will take a look at that option

Reply:
I’m using a LaunchDaemons because I’m running managedsoftwareupdate and deleting the flag file